server/trace/mock_trace.go

```diff
@@ -65,3 +68,10 @@ func (s *MockTraceServer) GetNumSpans(ctx context.Context, req *empty.Empty) (*m
 		NumSpans: int32(len(s.uploadedSpans)),
 	}, nil
 }
+
+func (s *MockTraceServer) SetDelay(ctx context.Context, req *mocktrace.SetDelayRequest) (*empty.Empty, error) {
```

Comment:
Exported functions should have a comment.

Can we add a CI step to run golint over the code? (not in this PR).

Reply:
Got it, added comments to all of the exported functions in `mock_trace.go` (there are probably others that we'll fix in a separate PR). Made an issue here: https://github.com/googleinterns/cloud-operations-api-mock/issues/27